### DISCUSSION TITLE
Analytic aspect of a problem

### DISCUSSION THREAD

**Discussion by blacktime1999** [EchterAlsFake/unofficial-api-for-pornhub#24] [Q&A] [answered]
![截图](https://github.com/Egsagon/PHUB/assets/95710926/89caa4e5-144f-41c0-83a5-1abc382d1d0c)
If a key is given normally, an error is reported！！！

**Comment by Egsagon** [comment, marked as answer]
Hey,
This looks like an old version of PHUB. You may want to update to the newest version (at least 4.1).

**Reply by blacktime1999** [reply]
thanks
